### DISCUSSION TITLE
How to arrange articles by date DESC?

### DISCUSSION THREAD

**Discussion by rastislavcore** [vuepress-theme-hope/vuepress-theme-hope#1915] [Q&A] [answered]
I couldn't find in the documentation the setting for the arrangement of the articles.

Is it possible to arrange articles by date DESC?

**Comment by Mister-Hope** [comment, marked as answer]
It seems that you are describing the default behavior.

Any links to demos where are article is not sorted by date DESC?

**Reply by rastislavcore** [reply]
Thank you, seems like that is the default, but how to change it? Do we have any option?

**Reply by Mister-Hope** [reply]
We provide sticky key for you to sticky articles, and the behavior is not customizable.

We can not make everything configurable

**Reply by Mister-Hope** [reply]
If you do have customization needs, you are free to describe it, and we will condidt if it's reasonable to add.
